microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -115,6 +115,25 @@ $ ip route
 ----
 This routing rule matches the Kubernetes service IP address range and forwards the packet to the gateway bridge `br-ex`. You must enable `ip_forward` on the host. After the packet is forwarded to the OVS bridge `br-ex`, it is handled by OpenFlow rules in OVS. OpenFlow then steers the packet to the OVN network and eventually to the pod.
 
+[id="microshift-4-14-notable-technical-changes"]
+== Notable technical changes
+
+{product-title} {product-version} introduces the following notable technical changes.
+
+// Note: use [discrete] for these sub-headings.
+
+[discrete]
+[id="microshift-4-14-networking-changes"]
+=== Networking changes
+
+Networking changes to {product-title} {product-version} include IP address assignment changes, gateway changes, and other refinements.
+
+[discrete]
+[id="microshift-4-14-cidr-removal"]
+==== CIDR removed
+
+CIDR notation for grouping IP addresses was removed with this release.
```

Comment:
Could you mention the `clusterNetwork` config where CIDR notation was removed? 
Just to make it clear on what user-facing config is updated.

Reply:
Yes, good note, thank you. Added detail and xref to the yaml example in the docs.